test/constrained-generics/basic-arg-query.good

```diff
@@ -0,0 +1,21 @@
+resolving interface declaration 'IFC'  basic-arg-query.chpl:20
+  required fn 'reqFun'
+  required fn 'dfltImpl'  with a default implementation
+
+resolving CG function early 'cgFun'  basic-arg-query.chpl:27
+
+resolving implements statement for ifc 'IFC'  basic-arg-query.chpl:37
+  witness 'reqFun'  basic-arg-query.chpl:33
+  default 'dfltImpl'  basic-arg-query.chpl:22
+
+checking constraint for 'IFC'  basic-arg-query.chpl:27
+  for call to cgFun  at basic-arg-query.chpl:39
+  satisfied with implements stmt  basic-arg-query.chpl:37
+
+considering CG candidate 'cgFun'  basic-arg-query.chpl:27
+  witness(es):
+  implements stmt for ifc 'IFC'  basic-arg-query.chpl:37
```

Comment:
These don't belong in the the .good file unless you add a developer-only flag to enable the tracing and provide it in the compopts.